crates/pop-contracts/src/build.rs

```diff
@@ -22,3 +30,21 @@ pub fn build_smart_contract(path: &Option<PathBuf>, build_release: bool) -> anyh
 
 	Ok(formatted_result)
 }
+
+/// Determines whether the manifest at the supplied path is a supported smart contract project.
+pub fn is_supported(path: Option<&Path>) -> Result<bool, Error> {
+	// Resolve manifest path
+	let path = match path {
+		Some(path) => match path.ends_with("Cargo.toml") {
```

Comment:
I see you're handling devs specifying the path to `Cargo.toml` like `cargo-contracts` does. I had this doubt at the beginning, and in the `path` description, I specified that it is the path to the project.
```rs
#[arg(short = 'p', long, help = "Path for the contract project, [default: current directory]")]
pub(crate) path: Option<PathBuf>,
```
It's great that you added this to handle both options seamlessly.

Reply:
This is mostly as the `cargo_toml::Manifest::from_path(..) `didnt work when using `"Cargo.toml",` as is output from `get_manifest_path` the path is `None`, but does work for "./Cargo.toml".